src/routing_node.rs

```diff
@@ -348,6 +352,11 @@ impl RoutingNode {
             return Err(RoutingError::BadAuthority);
         }
 
+        let (message, opt_token) = match self.accumulate(message_wrap) {
```

Comment:
Can we exclude `InternalResponse::CacheNetworkName` from this accumulator; we are currently not using `FindGroup`/`Response::FindGroup` so no need to mind that; for now just don't accumulate CacheNetworkName Responses